src/Hooks/hooks.h

```diff
@@ -0,0 +1,12 @@
+#ifndef HOOKS_H
+#define HOOKS_H
+
+#include "Hooks_global.h"
+
+class HOOKS_EXPORT Hooks
```

Comment:
I can almost guarantee that there will be no class called Hooks. Please remove it

Reply:
Removed Hooks.h and Hooks.cpp